source/blogs/posts/2017-02-25--wtf-is-sculpin.md

```diff
@@ -0,0 +1,206 @@
+
+* Writing content in markdown
+* TWIG templating
+* Syntax highlighting
+* Extremely fast
+* Secure (everything is client side)
+* Easily deployable (Can host static sites from github for free)
+
+When designing my blog I was looking for the path of least resistance for sharing some code snippets and jotting down a few of my opinions to share with the Interwebs. For someone that is already familiar with PHP Sculpin is the perfect tool to accomplish exactly this. How does one Sculpin?
+``
+## Installing Sculpin
+
+1. Download the starter kit
+
+```
+git clone https://github.com/sculpin/sculpin-blog-skeleton.git myblog
+cd myblog
+composer install
+```
+
+The starter kit is a skeleton build of Sculpin and is a great starting point for launching Sculpin as a blog.
+
+2. Run Sculpin
+
+```
+vendor/sculpin/sculpin/bin/sculpin generate --watch --server
+```
+
+Navigate to http://localhost:8000. BAM! You now have a boiler plate Sculpin site.
+
+## What's Happening?
+
+Magic! Okay maybe not magic. Sculpin comes with an executable bin, that we're running to parse all of the Twig templates, html, and markdown sources it can find to generate static html pages.
+
+The `--watch` simply tells Sculpin to watch for any file changes in order to rewrite any necessary HTML files. The `--server` launches PHP's built in web server which allows you to see your work in progress. The fact I don't need a working web server to get started makes it even easier to start creating content.
+
+## Creating Content in Sculpin
+
+Sculpin uses a markdown converter [michelf/php-markdown](https://github.com/michelf/php-markdown) for taking the blog content written in markdown and converting it to html. That means once the Twig templates and theme is in place, new content can be as simple as dropping a new markdown file into the code base. Luckily the starter kit comes with a working layout and theme, so we can get jump start to creating content.
+
+By default Sculpin will look for blog post content or **source objects** in each file under the `/source' directory. All content posts can be written in markdown which Sculpin with convert into html.
+
+```
+ # Markdown Cheatsheet
+ 
+ # This is an <h1>
+ ## This is an <h2>
+ ### This is an <h3>
+ 
+ We can place special emphasis on some words by making them **bold** or *italics*
+ 
+  Link
+  [More on Markdown](https://guides.github.com/features/mastering-markdown/)
+  
+ Lists
+ 1. Step 1
+ 1. ????
+ 1. Step 3: Profit
+ 
+ * Moon Knight
+ * Scarlet Which
+ * Gambit
+ * Hope Summers
+```
+ 
+ > # This is an &#60;h1&#62;
+ > ## This is an &#60;h2&#62;
+ > ### This is an &#60;h3&#62;
+ >
+ > We can place special emphasis on some words by making them **bold** or *italics*
+ >
+ > Link
+ > [More on Markdown](https://guides.github.com/features/mastering-markdown/)
+ 
+ > Lists
+ >
+ > 1. Step 1: Make numbered list
+ > 1. Step 2: ????
+ > 1. Step 3: Profit
+ > 
+ 
+* Moon Knight
+* Scarlet Which
```

Comment:
That's because the markdown converter won't interpret the <ul> markdown with in the blockquotes (the '>'. Did that to side step having to write a custom plugin.